data/test_config/config_ted_policy_model_checkpointing.yml

```diff
@@ -1,13 +1,7 @@
 language: "en"
-pipeline:
-  - name: WhitespaceTokenizer
-  - name: CountVectorsFeaturizer
-
 policies:
   - name: TEDPolicy
-    max_history: 5
-    epochs: 5
+    epochs: 2
     evaluate_on_number_of_examples: 14
     evaluate_every_number_of_epochs: 1
-    checkpoint_model: True
-  - name: RulePolicy
\ No newline at end of file
+    checkpoint_model: True
```

Comment:
can we please add empty lines at the end of all these test files due to [this](https://stackoverflow.com/a/2287990/3429596)?